src/BloomExe/FontProcessing/FontMetadata.cs

```diff
@@ -0,0 +1,351 @@
+﻿using System;
+using System.Linq;
+#if __MonoCS__
+using System.Diagnostics;
+using System.Text;
+using SharpFont;
+#else
+using System.Windows.Media;
+#endif
+
+namespace Bloom.FontProcessing
+{
+	public class FontMetadata
+	{
+		public string name { get; private set; }
+		public string version { get; private set; }
+		public string license { get; private set; }
+		public string licenseURL { get; private set; }
+		public string copyright { get; private set; }
+		public string manufacturer { get; private set; }
+		public string manufacturerURL { get; private set; }
+		public string fsType { get; private set; }
+		public string[] variants { get; private set; }
+		public string designer { get; private set; }
+		public string designerURL { get; private set; }
+		public string trademark { get; private set; }
+		public string determinedSuitability { get; private set; }
+		public string determinedSuitabilityNotes { get; private set; }
+
+		public FontMetadata(string name, FontGroup group)
+		{
+			this.name = name;
+#if __MonoCS__
```

Comment:
Worth a comment, like "GlyphTypeFace is better (why?) so we're using that for .net, but for mono we have to use SharpFont.